tutorial/managing-software.rst

```diff
@@ -0,0 +1,1611 @@
+this on one diagram, it would be quite extensive, so let's take a look through
+a single path.
+
+.. mermaid::
+
+    flowchart TD;
+      A[Ubuntu Package Archive] --> B([Splits into Ubuntu **series**]);
+
+      B --> C[e.g., mantic];
+      B --> D[noble]; 
+      B --> E[oracular, etc]; 
+
+      D --> H([Series split into **pockets**]);
+
+      H --> I[-release];
+      H --> J[-proposed];
+      H --> K[-updates];
+      H --> L[-security];
+      H --> M[-backports];
+
+      K --> N([Splits into **components**]);
+
+      N --> O[main];
+      N --> P[universe];
+      N --> Q[restricted];
+      N --> R[multiverse];
+
+      style C fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style D fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style E fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+
+      style I fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style J fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style K fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style L fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style M fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style O fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style P fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style Q fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+      style R fill:#fff,stroke:#cfcfcf,stroke-width:2px;
+
+Series
+^^^^^^
+
+The **series** is a set of packages that are released with a specific version
+of Ubuntu -- they're usually referred to by their codename (e.g., ``mantic``,
+``noble`` and ``oracular`` in our diagram). Each version of Ubuntu may have
+multiple releases (for example, an LTS will have an initial release when it
+launches (e.g. 24.04 LTS), and then "subsequent point releases" (e.g. 24.04.1
+LTS) -- these are all part of the same series (``noble``).
+
+In practice, people often use the term "Ubuntu release" and "Ubuntu series"
+interchangeably.
+
+Pockets
+^^^^^^^
+
+Every Ubuntu series (``noble``, ``jammy``, etc) is split into **pockets**,
+which are related to the software development/release lifecycle:
+
+- **-release** contains the packages as they are at release time.
+- **-proposed** contains package updates while they are being tested.
+- Once an update is released, they come from either **-security** or
+  **-updates** depending on whether they are a security-related update or not.
+- And **-backports**, which contains packages that were not available at release
+  time.
+
+This is why earlier, we saw that some updates came from ``noble-updates`` or
+``noble-security``. These refer to updates and security updates from the noble
+series (respectively). Pockets are usually appended to the end of the series,
+and it's quite common to see the hyphen (``-``) included when referring to
+pockets. 
+
+Remember that the original version of the ``apache2`` package that we saw came
+from ``noble`` -- the ``-release`` pocket only includes the software that was
```

Comment:
I didn't realise it would split that, never seen that happen before. I've tried to reword the paragraph a bit to avoid that word appearing at the end of the line, but it will depend on the screen size to some extent. At least on a regular monitor it shouldn't happen anymore.